dev-environment.yml

```diff
@@ -30,3 +30,6 @@ dependencies:
   - cython
   - panel
   - bokeh<2.3  # temp restriction until panel/bokeh bugs are worked out
+  - cmake >= 3.13.4
+  - conda-forge::ninja
+  - conda-forge::lit
```

Comment:
Same question here, but conda-forge has newer versions of `lit` than defaults.  Do we need one of those newer versions?  (In which case we should set a version constraint.)

Reply:
https://github.com/metagraph-dev/mlir-graphblas/pull/38/commits/053d56814992458df38906aa914637ea579ec2f6 removes the use of `conda-forge` for `lit`.